src/pages/validation-rules-analysis/ValidationRulesAnalysis.js

```diff
@@ -20,55 +21,116 @@ import { getDocsKeyForSection } from '../sections.conf';
 import { i18nKeys } from '../../i18n';
 
 // styles
-import pageStyles from '../PageStyles';
-import cssClasses from '../Page.css';
+import jsPageStyles from '../PageStyles';
+import cssPageStyles from '../Page.css';
+import ValidationRulesAnalysisTable from './validation-rules-analysis-table/ValidationRulesAnalysisTable';
+
 
 class ValidationRulesAnalysis extends Page {
+    constructor() {
+        super();
+
+        this.state = {
+            showTable: false,
+        };
+
+        this.validate = this.validate.bind(this);
+        this.back = this.back.bind(this);
+    }
+
+    validate() {
+        this.setState({ showTable: true });
+    }
+
+    back() {
+        this.setState({ showTable: false });
+    }
+
     render() {
         const translator = this.context.translator;
+        const elements = [];
+        let i = 0;
+        for (i; i < 55; i++) {
+            const one = {
+                label: i,
+                organisation: `Organisation ${i}`,
+                period: 'Mês do Ano X',
```

Comment:
??

Reply:
Just dummy information while we are not fetching data from backend. We will be populating it will real data soon.